tools/prepare_symlinks.yml

```diff
@@ -16,4 +16,6 @@
         mode: '0755'
         force: true
       loop: "{{ target_dirs.files }}"
-      when: item.path | basename | regex_search('^vmware_ops_')
+      when:
+        - item.path | basename | regex_search('^vmware_ops_')
+        - not item.path | basename | regex_search('vmware_ops_provision_virtual_esxi_test')
```

Comment:
why not vmware_ops_provision_virtual_esxi_test? 
Can you add a comment about it?

Reply:
the test is excluded because it has a unique `runme.sh` created manually. I added this comment to the `prepare_symlinks.yml` as well.
To run `vmware_ops_provision_virtual_esxi_test` on the simulator should also be run `mock_side_effects.yml` in parallel with the test, to power off the esxi hosts.